src/components/Header/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import {View, Text, Icon, Image} from 'react-native';
+import {Header} from 'react-native-elements';
+
+ScreenHeader = ({leftComponent, rightComponent}) => {
+  return (
+    <View>
```

Comment:
Do we really need a "View" component here to wrap the header component.?

Reply:
I think so it will work without View. I will remove it